src/store/participantsStore.js

```diff
@@ -247,23 +284,48 @@ const actions = {
 	},
 
 	async joinCall({ commit, getters }, { token, participantIdentifier, flags }) {
+		if (!participantIdentifier?.sessionId) {
+			console.error('Trying to join call without sessionId')
```

Comment:
no return, so we still want to attempt the server call?

Reply:
right, fixed